CenterofExcellenceAuditLogs/SolutionPackage/Other/Solution.xml

```diff
@@ -80,43 +80,54 @@
+      </MissingDependency>
+      <MissingDependency>
+        <Required type="EnvironmentVariableDefinition" displayName="admin_AdminMail" solution="CenterofExcellenceCoreComponents (2.5)" id.schemaname="admin_AdminMail" />
+        <Dependent type="29" displayName="[Child] Admin | Sync Logs" id="{751183fc-db68-ea11-a811-000d3a31cf04}" />
+      </MissingDependency>
+      <MissingDependency>
+        <Required type="EnvironmentVariableDefinition" displayName="admin_AdminMail" solution="CenterofExcellenceCoreComponents (2.5)" id.schemaname="admin_AdminMail" />
         <Dependent type="29" displayName="Admin | Sync Audit Logs" id="{fa91974d-6e68-ea11-a811-000d3a31cf04}" />
       </MissingDependency>
       <MissingDependency>
-        <Required type="EnvironmentVariableDefinition" displayName="admin_eMailHeaderStyle" solution="Active" id.schemaname="admin_eMailHeaderStyle" />
+        <Required type="EnvironmentVariableDefinition" displayName="admin_eMailHeaderStyle" solution="CenterofExcellenceCoreComponents (2.5)" id.schemaname="admin_eMailHeaderStyle" />
         <Dependent type="29" displayName="[Child] Admin | Sync Logs" id="{751183fc-db68-ea11-a811-000d3a31cf04}" />
       </MissingDependency>
       <MissingDependency>
-        <Required type="EnvironmentVariableDefinition" displayName="admin_eMailHeaderStyle" solution="Active" id.schemaname="admin_eMailHeaderStyle" />
+        <Required type="EnvironmentVariableDefinition" displayName="admin_eMailHeaderStyle" solution="CenterofExcellenceCoreComponents (2.5)" id.schemaname="admin_eMailHeaderStyle" />
         <Dependent type="29" displayName="Admin | Sync Audit Logs" id="{fa91974d-6e68-ea11-a811-000d3a31cf04}" />
       </MissingDependency>
       <MissingDependency>
-        <Required type="EnvironmentVariableDefinition" displayName="admin_PowerAutomateEnvironmentVariable" solution="Active" id.schemaname="admin_PowerAutomateEnvironmentVariable" />
+        <Required type="EnvironmentVariableDefinition" displayName="admin_PowerAutomateEnvironmentVariable" solution="CenterofExcellenceCoreComponents (2.5)" id.schemaname="admin_PowerAutomateEnvironmentVariable" />
         <Dependent type="29" displayName="[Child] Admin | Sync Logs" id="{751183fc-db68-ea11-a811-000d3a31cf04}" />
       </MissingDependency>
       <MissingDependency>
-        <Required type="EnvironmentVariableDefinition" displayName="admin_PowerAutomateEnvironmentVariable" solution="Active" id.schemaname="admin_PowerAutomateEnvironmentVariable" />
+        <Required type="EnvironmentVariableDefinition" displayName="admin_PowerAutomateEnvironmentVariable" solution="CenterofExcellenceCoreComponents (2.5)" id.schemaname="admin_PowerAutomateEnvironmentVariable" />
+        <Dependent type="29" displayName="Admin | Sync Audit Logs" id="{fa91974d-6e68-ea11-a811-000d3a31cf04}" />
```

Comment:
This was deleted from the solution. I guess it was probably moved to core...

Reply:
Yes, we are using this in Core as well now so took a dependency there! Thanks for checking!